lib/flight_scheduler/commands/run.rb

```diff
@@ -31,22 +31,30 @@ module FlightScheduler
   module Commands
     class Run < Command
       def run
+        create_con = connection.dup
+        create_con.params = { include: 'executions' }
```

Comment:
Because the following line calls for the `executions` which are a related resource:
https://github.com/openflighthpc/flight-scheduler/blob/8e7395bdb09fd285c4c303e22ffc7098179a9c04/lib/flight_scheduler/commands/run.rb#L59

The creation of the executions is some times sufficiently fast that they are returned with the initial `POST`. If not however, they need to be returned by the follow up `GET`.

Reply:
I'm with you.  The loop `until job_step.submitted` might never run because `job_step` might be `submitted` when returned from the `create` call.  In which case, we request the executions in the create to avoid making an additional GET.  Fair enough.